sqlmesh/core/table_diff.py

```diff
@@ -272,7 +272,22 @@ def _column_expr(name: str, table: str) -> exp.Expression:
             def name(e: exp.Expression) -> str:
                 return e.args["alias"].sql(identify=True)
 
-            query = (
+            source_query = (
+                exp.select(*(exp.column(c) for c in source_schema.keys()))
```

Comment:
you can just pass c, if it’s a string

you don’t need keys(), dict is iterable

Reply:
It looks like I still need to use `exp.column()` due to this effect:

```
>>> exp.select("country code").sql()
'SELECT country AS code'

>>> exp.select(exp.column("country code")).sql()
'SELECT "country code"'
```

however i've removed `.keys()`